mc.yaml

```
@@ -98,6 +98,52 @@ subpackages:
         - merged-usrsbin
         - wolfi-baselayout
 
+  - name: mc-iamguarded-2025-compat
+    description: "compat package with iamguarded/minio-client image"
+    dependencies:
+      provides:
+        - mc-iamguarded-compat=${{package.full-version}}
+      runtime:
+        - bash
+        - busybox
+        - coreutils
+        - merged-usrsbin
+        - wolfi-baselayout
+    pipeline:
+      - uses: iamguarded/build-compat
+        with:
+          package: minio-client
+          version: "2025"
+      - runs: |
+          mkdir -p /opt/iamguarded/minio-client/bin/
+          chmod g+rwX /opt/iamguarded
+
+          # Create symlink for mc binary
+          ln -sf /usr/bin/mc /opt/iamguarded/minio-client/bin/mc
+
+          # Create expected directories
+          mkdir -p /opt/iamguarded/minio-client/tmp
+          mkdir -p /opt/iamguarded/minio-client/logs
+          mkdir -p /.mc
```

```suggestion
          mkdir -p ${{targets.contextdir}}/.mc
```
because only `/opt/i` and `/i` are moving by finalize-compat